packages/contracts/test/helpers/DiamondTestHelper.sol

```diff
@@ -47,15 +46,15 @@ contract DiamondTestHelper is IDiamondCut, IDiamondLoupe, Test {
         uint256 index,
         bytes4[] memory array
     ) public pure returns (bytes4[] memory) {
-        bytes4[] memory newArray = new bytes4[](array.length - 1);
+        bytes4[] memory newarray = new bytes4[](array.length - 1);
```

Comment:
I cspell fixed this to newArray. Can you make sure to run cspell as part of your husky precommit hook? This shouldn't pass.

Reply:
yep fixed !